### PR TITLE
Secret documents can now be copied.

### PR DESCRIPTION
**Pull request by harmonyn**
- Secret documents can be copied in a photocopier. It uses 6 toner, one per document sheet.
- For the steal any documents objective, any copy will give you greentext.
- For the steal red/blue documents objective, a photocopy will not be accepted, as these original documents come with a seal. Clever traitors can take advantage of this by forging the seal with a red/blue crayon, resulting in documents that look normal unless closely examined.
- Cleaned up photocopier code slightly.

:cl:
rscadd: Due to budget cuts, Nanotrasen is no longer utilizing copy-protected paper for its classified documents. Fortunately, our world-class security team has always prevented any thefts or photocopies from being made!
rscadd: Secret documents can be photocopied. If you have an objective to steal any set of documents, a photocopy will be accepted. If you must steal red or blue documents, a photocopy will NOT be accepted. Enterprising traitors can forge the red/blue seal with a crayon to take advantage of this.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/photocopier.dm [hunk 1]
```diff
@@ -116,6 +117,17 @@
 					busy = 0
 				else
 					break
+		else if(doccopy)
+			for(var/i = 0, i < copies, i++)
+				if(toner > 0 && !busy && doccopy)
```
Comment (KazeEspada) [inline code/modules/paperwork/photocopier.dm]: toner should be greater than 5, since you are about to subtract by 6.

**Comment by Cheridan** [comment]
so cool

**Comment by Ultimate-Chimera** [comment]
:+1:

**Comment by Dorsisdwarf** [comment]
Doesn't taking a photo do this already?

**Comment by bgobandit** [comment]
You're thinking of the blueprints.

**Pull request merged**